docs/overview/stakeholder.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/trustbloc/trustbloc-did-method/docs/overview/stakeholder.schema.json",
```

Comment:
should this be member.schema.json to match with the other file property name?

Reply:
(basically use the more generic member form anywhere there might be a permanent key/name.)